2-mandatory/1-weather-report.js

```diff
@@ -12,52 +12,50 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+  let newArray = [];
```

Comment:
It's best to try and use variable names that are descriptive. Here, `temperatureReports` would be more suitable than `newArray`